ui/src/org/pentaho/di/ui/trans/steps/ivwloader/IngresVectorwiseLoaderDialog.java

```diff
@@ -404,9 +403,7 @@ public void widgetDefaultSelected( SelectionEvent event ) {
   }
 
   protected TextVar addPasswordTextVar( String labelMessageKey, Control prevControl ) {
-    TextVar textVar = addStandardTextVar( labelMessageKey, prevControl );
```

Comment:
I've removed the addPasswordTextVar method from the step dialog, as it was not used anywhere in the class, package, or project itself.  The only password in this step is done via the Database Connection meta.